Detectors/FIT/FT0/simulation/src/Detector.cxx

```diff
@@ -192,14 +192,14 @@ void Detector::ConstructGeometry()
 
     //   com[itr-Geometry::NCellsA] = new TGeoCombiTrans(tr[itr],rot[itr-Geometry::NCellsA]);
     com[ic] = new TGeoCombiTrans(xc2[ic], yc2[ic], (zc2[ic] - 80), rot[ic]);
+    std::cout << ic << " " << xc2[ic] << " " << yc2[ic] << std::endl;
```

Comment:
@AllaMaevskaya It is difficult to track such anonymous printouts in the log, this looks like a debug output not removed by accident. If it should go to the log, please add some meaningful message and better use ``LOG(INFO)``

Reply:
OK, done